packages/tools/u-boot/package.mk

```diff
@@ -46,9 +70,18 @@ make_target() {
   else
     [ "${BUILD_WITH_DEBUG}" = "yes" ] && PKG_DEBUG=1 || PKG_DEBUG=0
     [ -n "$ATF_PLATFORM" ] &&  cp -av $(get_build_dir atf)/bl31.bin .
-    DEBUG=${PKG_DEBUG} CROSS_COMPILE="$TARGET_KERNEL_PREFIX" LDFLAGS="" ARCH=arm make mrproper
-    DEBUG=${PKG_DEBUG} CROSS_COMPILE="$TARGET_KERNEL_PREFIX" LDFLAGS="" ARCH=arm make $($ROOT/$SCRIPTS/uboot_helper $PROJECT $DEVICE $UBOOT_SYSTEM config)
-    DEBUG=${PKG_DEBUG} CROSS_COMPILE="$TARGET_KERNEL_PREFIX" LDFLAGS="" ARCH=arm _python_sysroot="$TOOLCHAIN" _python_prefix=/ _python_exec_prefix=/ make HOSTCC="$HOST_CC" HOSTLDFLAGS="-L$TOOLCHAIN/lib" HOSTSTRIP="true" CONFIG_MKIMAGE_DTC_PATH="scripts/dtc/dtc"
+    case "${UBOOT_SYSTEM:-$PROJECT}" in
+      odroid-n2)
```

Comment:
I am not a very big fan of this old "odroid-c2" hack, as long as there is a plan to revert it once support for odroid-n2 lands in u-boot mainline I am okay with this. Is special u-boot even needed for the n2?, don't the n2 come with bootable spi flash we can use?

Reply:
I have no idea. I will leave this patch for a PR from someone who actually owns the device.

Reply:
Yes, for N2 it is now possible to run all systems using u-boot in SPI. Assembly and entry in the image LE version of u-boot for N2 is not required. It remains only to wait for the representatives of HK to build a new version of u-boot for SPI with support for multi-boot, or it is necessary to crash a new version of such an update on their own (I'm trying to figure out how to build a new version of the update for SPI). User can now activate the multi-boot to SPI using a UART console. I checked, the LE image works well with the new startup version without the built-in u-boot from any media. Including USB 3.0

Reply:
What does SPI boot look for on media? `boot.ini` or something else?

Reply:
To run from USB while using your "boot.scr", two variants of "boot" are used for SD\eMMC.ini" and 'boot.scr." The startup sequence can be changed as convenient to the user. By default, the order is : USB, SD, eMMC RAWIMAGE (I understand this is for Android) PXE Preboot.